source/systems/hera_user_guide.rst

```diff
@@ -22,6 +22,16 @@ There are currently two major systems at NESCC:
 - HPSS* A 50 Petabyte IBM/Oracle hierarchical storage management
   system.
 
+In 2025, two systems are scheduled for addition to NESCC.
```

Comment:
Looking at the processed file this prosaic paragrah doesn't look very pleasing, especially when there is a bulleted list just above that about Hera and HPSS.  Using the same style may make it look better.

In 2025, two systems are scheduled for addition to NESCC:
- Ursa, based on AMD 9654 and DDN Lustre, should be in place early in 2025
- Rhea, based on AMD Turin and DDN Lustre & VAST File Systems, is expected in the second half of the calendar year

In first set of bullets for Hera and HPSS, one has a period and the other one doesn't, might be good to make them consistent.

Reply:
I love a picky edit -- keep 'em coming.
Done, thanks Raghu.